app/Http/Requests/StoreVolumeReport.php

```diff
@@ -119,8 +122,13 @@ public function withValidator($validator)
      */
     public function getOptions()
     {
-        return array_merge(parent::getOptions(), [
+        \Log::info('StoreVolumeReport input:', $this->all());
```

Comment:
Please don't use `Log` anywhere. We only log exceptions and that's done automatically.